helm-charts/splunk-otel-collector/templates/clusterRole.yaml

```diff
@@ -14,6 +14,16 @@ rules:
   verbs:          ['use']
   resourceNames:  [{{ template "splunk-otel-collector.fullname" . }}]
 {{- end }}
+{{- if .Values.otelK8sClusterReceiver.k8sEventsEnabled }}
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - get
+  - update
+  - patch
```

Comment:
I don't think we need patch and update verbs on configmaps. Could you please remove them and test? It might be even the case that no access to configmaps needed at all